.github/actions/build-image-action/action.yml

```diff
@@ -45,6 +45,9 @@ inputs:
   runs_on:
     description: "The runner to use"
     required: true
+  build_target:
```

Comment:
consider need this? or just use model name - its only different for one

Reply:
Removed